spec/models/issue_tta_report_spec.rb

```diff
@@ -5,8 +5,7 @@
     subject do
       described_class.new(
         issue_id: "ISSUE_ID",
-        tta_report_display_id: "ACTIVITY_REPORT_ID",
-        tta_report_id: "12345"
```

Comment:
[suggestion] i think we need another comment here about how it is retrieving the tta_report_id based on the display_id regex match, as i again didn't get it until i looked more closely at the test data.